### DISCUSSION TITLE
Schedule not running for Github Managed App?

### DISCUSSION THREAD

**Discussion by broksonic21** [renovatebot/renovate#9321] [Request Help]
All of our projects missed their scheduled times today as RenovateBot was running only every 5-10 hours depending on the project, per the logs. These are active projects that normally have renovate check every hour or so (though the actual PR creation is time limited). Is something going on with the schedules?

**Comment by viceice** [comment]
Github managed app is disabled, see https://docs.renovatebot.com/install-gitlab-app/ for more information 

See following to run renovate self-hosted
https://gitlab.com/renovate-bot/renovate-runner

**Reply by rarkins** [reply]
GitHub though, not GitLab?

**Comment by viceice** [comment]
> GitHub though, not GitLab?

Sorry, mistake. 🤦‍♂️
